### PR TITLE
Add Ξcc workflows. Add analysis scripts.

### PR DESCRIPTION
**Pull request by ginnocen**
To be merged after https://github.com/AliceO2Group/AliceO2/pull/6843

### CODE REVIEW

File: codeHF/workflows.yml [hunk 1]
```diff
@@ -28,6 +28,16 @@ workflows:
       default: [HFCANDP3BASE, HFCANDP3EXT]
       mc: [HFCANDP3MCREC, HFCANDP3MCGEN]
 
+  o2-analysis-hf-candidate-creator-xicc:
+    <<: *cand_creator
+    activate: no
+    dependencies: [o2-analysis-hf-xic-topkpi-candidate-selector]
+    options:
+      mc: "--doMC"
```
Comment (vkucera) [inline codeHF/workflows.yml:36]: Needless to specify this as it is already inherited from `cand_creator`.

File: codeHF/workflows.yml [hunk 2]
```diff
@@ -50,6 +60,11 @@ workflows:
     <<: *selector_3prong
     activate: no
 
+  o2-analysis-hf-xicc-topkpipi-candidate-selector:
+    <<: *selector_2prong
```
Comment (vkucera) [inline codeHF/workflows.yml:64]: Inheriting from `selector_2prong` seems useless here if all the attributed are overwritten.

**Review by vkucera — CHANGES_REQUESTED** [review]
Shouldn't the other JSON files be updated too?

File: codeHF/config_tasks.sh [hunk 3]
```diff
@@ -40,17 +40,21 @@ DOO2_PID_TOF_QA=0   # pid-tof-qa-mc
 DOO2_SKIM=0         # hf-track-index-skims-creator
 DOO2_CAND_2PRONG=0  # hf-candidate-creator-2prong
 DOO2_CAND_3PRONG=0  # hf-candidate-creator-3prong
+DOO2_CAND_XICC=0    # hf-candidate-creator-xicc
 # Selectors
 DOO2_SEL_D0=0       # hf-d0-candidate-selector
 DOO2_SEL_DPLUS=0    # hf-dplus-topikpi-candidate-selector
 DOO2_SEL_LC=0       # hf-lc-candidate-selector
 DOO2_SEL_XIC=0      # hf-xic-topkpi-candidate-selector
+DOO2_SEL_XICC=0     # hf-xic-topkpipi-candidate-selector
```
Comment (vkucera) [inline codeHF/config_tasks.sh:49]: `xic` -> `xicc`

**Comment by ginnocen** [comment]
thanks a lot @vkucera for the comments.

File: codeHF/dpl-config_run5_hf.json [hunk 4]
```diff
@@ -237,16 +248,84 @@
...
+                [ "0.300", "0.3", "0.3", "0.3", "1.e-2", "0.005", "0.3"],
+                [ "0.300", "0.4", "0.4", "0.4", "1.e-2", "0.005", "0.3"],
+                [ "0.300", "0.4", "0.4", "0.4", "1.e-2", "0.005", "0.3"],
+                [ "0.300", "0.4", "0.4", "0.4", "1.e-2", "0.005", "0.3"],
+                [ "0.300", "0.4", "0.4", "0.4", "1.e-2", "0.005", "0.3"],
+                [ "0.300", "0.4", "0.4", "0.4", "1.e-2", "0.005", "0.3"]
+            ]
+        }
+    },
+    "hf-xicc-topkpipi-candidate-selector": {
```
Comment (vkucera) [inline codeHF/dpl-config_run5_hf.json:291]: This does not seem to match the O2 device name.

File: codeHF/config_input.sh [hunk 5]
```diff
@@ -4,17 +4,17 @@
...
 
 # Number of input files per job (Automatic optimisation on if < 1.)
 NFILESPERJOB_CONVERT=0  # Conversion
 NFILESPERJOB_ALI=0      # AliPhysics
-NFILESPERJOB_O2=1       # O2
+NFILESPERJOB_O2=2       # O2
 
 # Maximum number of simultaneously running O2 jobs
-NJOBSPARALLEL_O2=$(python3 -c "print(min(10, round($(nproc) / 2)))")
+NJOBSPARALLEL_O2=$(python3 -c "print(min(20, round($(nproc) / 2)))")
```
Comment (vkucera) [inline codeHF/config_input.sh:17]: Please revert unnecessary changes.

File: codeHF/config_tasks.sh [hunk 6]
```diff
@@ -40,17 +40,21 @@ DOO2_PID_TOF_QA=0   # pid-tof-qa-mc
...
+DOO2_CAND_XICC=0    # hf-candidate-creator-xicc
 # Selectors
 DOO2_SEL_D0=0       # hf-d0-candidate-selector
 DOO2_SEL_DPLUS=0    # hf-dplus-topikpi-candidate-selector
 DOO2_SEL_LC=0       # hf-lc-candidate-selector
-DOO2_SEL_XIC=0      # hf-xic-topkpi-candidate-selector
+DOO2_SEL_XIC=1      # hf-xic-topkpi-candidate-selector
+DOO2_SEL_XICC=1     # hf-xicc-topkpipi-candidate-selector
 DOO2_SEL_JPSI=0     # hf-jpsi-toee-candidate-selector
+
```
Comment (vkucera) [inline codeHF/config_tasks.sh:51]: Please revert unnecessary changes.

File: codeHF/config_tasks.sh [hunk 7]
```diff
@@ -210,6 +224,7 @@ function MakeScriptPostprocess {
     [ $DOO2_TASK_DPLUS -eq 1 ] && PARTICLES+="-dplus"
     [ $DOO2_TASK_LC -eq 1 ] && PARTICLES+="-lc"
     [ $DOO2_TASK_XIC -eq 1 ] && PARTICLES+="-xic"
+    [ $DOO2_TASK_XICC -eq 1 ] && PARTICLES+="-xicc"
```
Comment (vkucera) [inline codeHF/config_tasks.sh:227]: The strings now use spaces as separators.

File: codeHF/workflows.yml [hunk 8]
```diff
@@ -28,6 +28,14 @@ workflows:
       default: [HFCANDP3BASE, HFCANDP3EXT]
       mc: [HFCANDP3MCREC, HFCANDP3MCGEN]
 
+  o2-analysis-hf-candidate-creator-xicc:
+    <<: *cand_creator
+    activate: no
+    dependencies: [o2-analysis-hf-xic-topkpi-candidate-selector]
```
Comment (vkucera) [inline codeHF/workflows.yml:34]: `activate` is obsolete.
Single dependency can be a simple string (instead of a list).

**Review by vkucera — CHANGES_REQUESTED** [review]
Please resolve the conflicts and update the files.

**Pull request closed without merging**